content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -16,7 +16,7 @@ Follow these instructions to prepare an OpenShift cluster for Istio.
 Install Istio using the OpenShift profile:
 
 {{< text bash >}}
-$ istioctl install --set global.platform=openshift
+$ istioctl install --set profile=openshift
```

Comment:
This should be `istioctl install --set values.global.platform=openshift` - using the Helm values.

istioctl-only profiles via `--set profile` are/should be deprecated, and `openshift` is now a `platform`, and not a `profile`.

This is so we can compose like so: `istioctl install --set values.global.platform=openshift  --set values.global.profile=ambient`

Reply:
Okay Ben. We can continue with your PR - https://github.com/istio/istio.io/pull/16051/files